backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/translations/v2TranslationsController/TranslationsControllerFilterTest.kt

```diff
@@ -44,6 +44,23 @@ class TranslationsControllerFilterTest : ProjectAuthControllerTest("/v2/projects
     }
   }
 
+  @ProjectJWTAuthTestMethod
+  @Test
+  fun `filters by complex keyName with dot`() {
```

Comment:
Thx! Well, it's breaking change anyway, but I don't think anyone used the coma separated value in this case.
 
> API-wide through FormatterRegistry configuration

In different endpoints, this might be used, so I don't want to change this, now. Maybe in API `v3` 😀